projects/Mallard/src/components/article/html/components/header.ts

```diff
@@ -576,28 +581,88 @@ const isImmersive = (type: ArticleType) =>
     type === ArticleType.Obituary ||
     type === ArticleType.Gallery
 
-const hasLargeByline = (type: ArticleType) =>
-    type === ArticleType.Opinion || type === ArticleType.Analysis
-
-const Header = ({
-    publishedId,
-    type,
-    getImagePath,
-    ...headerProps
-}: {
-    showMedia: boolean
-    publishedId: Issue['publishedId'] | null
-    type: ArticleType
-    canBeShared: boolean
-    getImagePath: GetImagePath
-} & ArticleHeaderProps) => {
-    const immersive = isImmersive(type)
-    const largeByline = hasLargeByline(type)
+const setStandFirst = (
+    articleHeaderType: HeaderType,
+    type: ArticleType,
+    headerProps: ArticleHeaderProps,
+    publishedId: Issue['publishedId'] | null,
+    getImagePath: GetImagePath,
+): string => {
     const cutout =
         type === ArticleType.Opinion &&
         headerProps.bylineImages &&
         headerProps.bylineImages.cutout
-    const shareButton = !headerProps.canBeShared
+    if (articleHeaderType === HeaderType.LargeByline) {
+        return html`
+            <section class="header-top">
+                <div class="${cutout && `header-opinion-flex`}">
+                    <h1>
+                        ${type === ArticleType.Opinion && Quotes()}
+                        <span class="header-top-headline"
+                            >${headerProps.headline}
+                        </span>
+                        <span class="header-top-byline"
+                            >${headerProps.bylineHtml}
+                        </span>
+                    </h1>
+                    ${publishedId &&
+                        cutout &&
+                        html`
+                            <div>
+                                ${Image({
+                                    image: cutout,
+                                    getImagePath,
+                                })}
+                            </div>
+                        `}
+                </div>
+            </section>
+        `
+    } else {
+        return html`
+            <section class="header-top">
+                <h1>
+                    ${headerProps.headline}
+                </h1>
+                ${articleHeaderType === HeaderType.RegularByline
+                    ? `<p>
+                        ${headerProps.standfirst}
+                      </p>`
+                    : ``}
```

Comment:
an alternative syntax which seems to be the convention for this kind of situatio in js templates is like this - I'm not convinced it's more readable though so up to you! 

```
${articleHeaderType === HeaderType.RegularByline && `<p>
                        ${headerProps.standfirst}
                      </p>`}
```